reviewqueue/helpers.py

```diff
@@ -105,6 +105,8 @@ def charmstore_login(settings):
         token_value = settings.get('charmstore.usso_token')
         if not token_value:
             raise ValueError('Missing USSO token')
+        if not os.path.isdir(os.path.dirname(token_path)):
+            os.makedirs(os.path.dirname(token_path))
```

Comment:
`os.makedirs()` could raise OSError if the dirs were to be created by another thread after the `isdir()` check.